train.py

```diff
@@ -372,6 +373,7 @@ def loss_fn(pred, labels):
                     "loss_metrics/global_avg_loss": global_avg_loss,
                     "loss_metrics/global_max_loss": global_max_loss,
                     "throughput(tps)": tps,
+                    "tflops/s": tflops,
```

Comment:
```suggestion
                    "tflops(s)": tflops,
```
otherwise it will likely go to a subfolder

Reply:
thanks!!